.github/workflows/ci.yml

```diff
@@ -51,3 +51,18 @@ jobs:
 
       - name: Build SATySFi
         run: opam exec -- make all
+
+      - name: Run tests
+        run: |
+          opam exec -- dune runtest
+
+      - name: Install SATySFi
+        run: |
+          opam install -y .
+          ./download-fonts.sh
```

Comment:
Can you cache the downloaded fonts (see https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows)?  The other option is to have OPAM cache downloaded files (#149).

When I was using Travis CI, running download-fonts.sh without caching had the CI instances banned from some font distributing sites.

Reply:
Thank you. I have added cache settings.

Reply:
Thank you!